README.md

```diff
@@ -49,13 +50,22 @@ A curated list of open source projects used in nuclear science and engineering.
 
 - [Nek5000](https://github.com/Nek5000/Nek5000) — Spectral-element CFD code
 - [OpenFOAM](https://www.openfoam.com/) — Finite volume CFD code
+- [nekRS](https://github.com/Nek5000/nekRS) — MPI+X hybrid parallelism Spectral-element CFD code supporting CPU, CUDA, HIP and OpenCL
 
 ## Multiphysics
 
 - [MOOSE](https://github.com/idaholab/moose) — Finite-element, multiphysics framework
 - [Okapi](https://github.com/aprilnovak/okapi) — OpenMC wrapped as a MOOSE app
 - [ENRICO](https://github.com/enrico-dev/enrico) — Monte Carlo + CFD coupling application
 
+## Molten Salt Reactor
+
+- [moltres](https://github.com/arfc/moltres) — A molten salt reactor simulator code
+
+## Advanced Reactor and Fuel Cycle Research Group
```

Comment:
```suggestion
##  Research Groups Invested in Open Source Tools for Nuclear Science and Engineering
```
This will allow other groups to add themselves.  It might make more sense to put this after the "Other" category